daily_milestone.go

```diff
@@ -250,7 +253,7 @@ func main() {
 	var advance int
 	// Command Line Parsing Starts
 	flag.StringVar(&token, "Token", "jGWPwqQUuf37b", "Gitlab api key/token.")
-	flag.StringVar(&timeInterval, "TimeInterval", "daily", "to set milestone to daily, weekly or monthly.")
+	flag.StringVar(&timeInterval, "TimeInterval", "daily", "Set milestone to daily, weekly or monthly.")
```

Comment:
Only thing left for consistency reasons is removing the "." on any of these descriptions.